Header header
detection_object[] detection_objects
uint32 frame_det_id
string seq_id